lib/views/pages/profile/profile_page.dart

```diff
@@ -65,12 +66,18 @@ class ProfilePage extends HookConsumerWidget {
             appBar: AppBar(
               leading: const LeadingBackButton(),
               title: UserAccount(
-                userEmail: value.user.email ?? "default@mail.com",
+                userName: value.firestoreUser.data.username,
+                userDisplayName: value.firestoreUser.data.displayName,
+                centauriPoints: value.firestoreUser.data.centauriPoints,
               ),
               actions: [
                 IconButton(
+                  key: settingsKey,
                   icon: const Icon(Icons.settings),
-                  onPressed: () {},
+                  onPressed: () {
+                    //add 5 points to the user
+                    ref.read(profileProvider.notifier).addPoints(5);
```

Comment:
This code should not be merged with the main branch, as we don't want test code to stay in the UI.
I don't think we want to have a button that just gives 5 points to any user that presses it (for any amount of times).